go-controller/pkg/node/default_node_network_controller.go

```diff
@@ -376,6 +377,69 @@ type managementPortEntry struct {
 	config *managementPortConfig
 }
 
+// getEnvNameFromResourceName gets the device plugin env variable from the device plugin resource name.
+func getEnvNameFromResourceName(resource string) string {
+	res1 := strings.ReplaceAll(resource, ".", "_")
+	res2 := strings.ReplaceAll(res1, "/", "_")
+	return "PCIDEVICE_" + strings.ToUpper(res2)
+}
+
+// getDeviceIdsFromEnv gets the list of device IDs from the device plugin env variable.
+func getDeviceIdsFromEnv(envName string) ([]string, error) {
+	envVar := os.Getenv(envName)
+	if len(envVar) == 0 {
+		return nil, fmt.Errorf("unexpected empty env variable: %s", envName)
+	}
+	deviceIds := strings.Split(envVar, ",")
+	return deviceIds, nil
+}
+
+// retrieveDevicePluginResources tries to retrieve any device plugin resources passed in via arguments and device plugin env variables.
+func retrieveDevicePluginResources() error {
+	if config.OvnKubeNode.MgmtPortDPResourceName != "" {
```

Comment:
Can we use early return?

Reply:
I think your second comment address this as well.